src/milia/api/organization.cljc

```diff
@@ -107,6 +108,26 @@
                  :suppress-4xx-exceptions? true
                  :as-map? true))))
 
+(defn change-org-member-role
+  "Change the role of an organization member"
+  [member org-name event-chan add-callback?]
+  (let [data {:username (:username member)
+              :role (:role member)}]
+    (apply parse-http (->
+                        [:put
+                         (make-url "orgs" org-name "members")
+                         :http-options
+                         #?(:clj   {:form-params data
+                                    :content-type :json})
+                         #?(:cljs {:json-params data})
+                          :as-map? true]
+                        (concat (when add-callback?
+                                  [:callback
+                                   #?(:cljs
+                                    #(put! event-chan
+                                           {:updated-member member}))]))
+                        (vec)))))
```

Comment:
is `vec` required?

Reply:
Yes because `concat` returns a `seq` and i need a `vector` for the `apply`.